alibaba-dragonwell/content.md

```diff
@@ -0,0 +1,42 @@
+## Overview
+
+The images in this repository contain Alibaba Dragonwell binaries built by [JDK Team at Alibaba Cloud](https://github.com/alibaba/dragonwell8).
+
+# What is Alibaba Dragonwell ?
+
+Alibaba Dragonwell, as a downstream version of OpenJDK, is the OpenJDK implementation at Alibaba optimized for online e-commerce, financial, and logistics applications running on 100,000+ servers. Alibaba Dragonwell is the engine that runs these distributed Java applications in extreme scaling.
+
+# Do you provide JRE (Java Runtime Environment) Images?
+
+Current there are only JDK (Java Developer Kit) images.
+
+# How to use this Image
+
+To run a pre-built jar file with the latest OpenJDK 11, use the following Dockerfile:
+
+```dockerfile
+FROM %%IMAGE%%:11
+RUN mkdir /opt/app
+COPY japp.jar /opt/app
+CMD ["java", "-jar", "/opt/app/japp.jar"]
+```
+
+You can build and run the Docker Image as shown in the following example:
+
+```console
+docker build -t japp .
+docker run -it --rm japp
+```
+
+### Using a different base Image
+
+If you are using a distribution that we don't provide an image, you can copy the JDK using a similar Dockerfile to the
+one below:
+
+```dockerfile
+# Example
+FROM <base image>
+ENV JAVA_HOME=/opt/java/openjdk
```

Comment:
Just a note: this is officially documenting `/opt/java/openjdk` as an "interface" provided by the image, which means it will be harder to change in the future if you ever decide that directory *shouldn't* be where the JDK is installed.

Reply:
Yes.  I think we will use `/opt/java/openjdk` just to be more `compatible` to openjdk images.